daemon/control.c

```diff
@@ -32,6 +32,7 @@
 #include "cmld.h"
 #include "hardware.h"
 #include "crypto.h"
+#include "uevent.h"
```

Comment:
uevent.h not used by this commit! remove. (merge error?!)

Reply:
Fixed